pkg/controller/nmstate/nmstate_suite_test.go

```diff
@@ -9,6 +12,68 @@ import (
 	"github.com/onsi/ginkgo/reporters"
 )
 
+func copyFile(src, dst string) error {
+	var err error
+	var srcfd *os.File
+	var dstfd *os.File
+	var srcinfo os.FileInfo
+
+	_, fileName := filepath.Split(src)
+	dst = dst + fileName
+	if srcfd, err = os.Open(src); err != nil {
+		return err
+	}
+	defer srcfd.Close()
+
+	if dstfd, err = os.Create(dst); err != nil {
+		return err
+	}
+	defer dstfd.Close()
+
+	if _, err = io.Copy(dstfd, srcfd); err != nil {
+		return err
+	}
+	if srcinfo, err = os.Stat(src); err != nil {
+		return err
+	}
+	return os.Chmod(dst, srcinfo.Mode())
+}
+
+func copyManifests() error {
+	dirArr := []string{"crds/", "namespace/", "handler/", "rbac/"}
+	srcToDest := map[string]string{
+		"../../../deploy/crds/nmstate.io_nodenetworkconfigurationenactments_crd.yaml": "./testdata/kubernetes-nmstate/crds/",
+		"../../../deploy/crds/nmstate.io_nodenetworkconfigurationpolicies_crd.yaml":   "./testdata/kubernetes-nmstate/crds/",
+		"../../../deploy/crds/nmstate.io_nodenetworkstates_crd.yaml":                  "./testdata/kubernetes-nmstate/crds/",
+		"../../../deploy/handler/namespace.yaml":                                      "./testdata/kubernetes-nmstate/namespace/",
+		"../../../deploy/handler/operator.yaml":                                       "./testdata/kubernetes-nmstate/handler/handler.yaml",
+		"../../../deploy/handler/service_account.yaml":                                "./testdata/kubernetes-nmstate/rbac/",
+		"../../../deploy/handler/role.yaml":                                           "./testdata/kubernetes-nmstate/rbac/",
+		"../../../deploy/handler/role_binding.yaml":                                   "./testdata/kubernetes-nmstate/rbac/",
+	}
+	for _, dir := range dirArr {
```

Comment:
We can move this to `copyFile` extracting path from destination file no problem on calling MkdirAll if we have already created the dir.

Reply:
Yep, good idea, I'll change that.